CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 * Dependency updates (Kafka 3.6.1, Kubernetes configuration provider 1.1.2, Vert.x 4.5.2, Netty 4.1.106.Final, Jackson FasterXML 2.16.1, Micrometer 1.12.2)
 * Fixed missing messaging semantic attributes to the Kafka consumer spans
+* Removed the dependency on OkHttp and thus Kotlin.
+* Dependency update (OpenTelemetry Semconv from 1.19.0-alpha to 1.21.0-alpha)
```

Comment:
I guess we are missing all the other bumps you made in the pom.
Also it's enough just naming component - version so for example ...

```shell
Dependency update (OpenTelemetry Semconv 1.21.0-alpha, ....)
```

Reply:
This probably should also not have its own line in the changelog but be added to the existing dependency updates?

Reply:
Yeah right.

Reply:
I've added Semconv to the global dependency list, but thought it was worth calling out this particular change as it leads to the changes of the tags included on spans. 

I can re-phrase if that is desired but I think mentioning the changes is merited.

Reply:
If it is for whatever reason more important, you should probably rephrase it then:
* It doesn't make sense to have to separate entries for `Dependency update (...)`
* If this is more important for some reason, explain it there. If it changes some tags, make it clear there what and how.

You seem to have the list of changes already, so you can just say something like 
```
* OpenTelemetry Semconv update causes following incompatible changes:
    * ...
```